docs/reference/scripting.asciidoc

```diff
@@ -4,61 +4,76 @@
 [partintro]
 --
 With scripting, you can evaluate custom expressions in {es}. For example, you
-could use a script to return "script fields" as part of a search request or
+could use a script to return fields as part of a search request or
```

Comment:
You can already return fields as part of the request, why would ya need a script?  
Perhaps something like: 
>"could use a script to return a computed value as a field"